pc_ble_driver_py/ble_driver.py

```diff
@@ -1540,6 +1545,11 @@ def from_c(cls, gattc_char):
             handle_decl=gattc_char.handle_decl,
             handle_value=gattc_char.handle_value,
         )
+    
+    def __eq__(self, other):
+        if isinstance(other, BLECharacteristic):
+            return self.uuid == other.uuid
+        return self.uuid == other
```

Comment:
Thanks for making the change.  I think your merge from your own master got messed up, and it reintroduced the __eq__functions